org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -83,6 +83,33 @@ class Translator {
     }
 
     /**
+     * Subscribe to notifications for a platform.
+     * This function is intended to be called by the platform translator for initial subscription,
+     * and on the hub translator (this) for verification.
+     */
+    postSubscribe(subscriptionInfo) {
```

Comment:
We don't add platform specific structure to the RAML because it is only used internally when calling this function from the platform translator.  postSubscribe(...) on a hub only needs to be exposed for verification (which isn't applicable for smartthings, but must still be exposed as it's in the RAML.  In this case, it is what is described in the RAML, it contains a callbackURL.

---
In reply to: [92070425](https://github.com/openT2T/translators/pull/136#discussion_r92070425) [](ancestors = 92070425)